using_images/s2i_images/ruby.adoc

```diff
@@ -119,13 +121,11 @@ Your application needs to be built with one of gems that reloads the server ever
 
 Please note that in order to be able to run your application in development mode, you need to modify the link:https://github.com/openshift/source-to-image#anatomy-of-a-builder-image[S2I run script], so the web server is launched by the chosen gem, which checks for changes in the source code.
 
-After you built your application image with your version of link:#s2i-scripts[S2I run script], run the image with the RACK_ENV=development environment variable passed to the link:http://docker.io[Docker] -e run flag:
+After you built your application image with your version of link:#s2i-scripts[S2I run script], run the image with the RACK_ENV=development environment variable. See for example link:dev_guide/new_app.html#specifying-environment-variables[oc new-app].
```

Comment:
I didn't include the `oc new-app` example, because I think that you will usually create a "normal" application and change the environment variable if and when you need to debug something in your application. That's why I included an `oc env` example. I've added the `oc new-app` link for the use case that the developer is developing his application with OC and wants to use development mode always.